estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_estate_model,access_estate_model,model_estate_property,base.group_user,1,1,1,1
```

Comment:
The name of the model is always separated with dots instead of underscore
```suggestion
access_estate_model,access.estate.model,model_estate_property,base.group_user,1,1,1,1
```